CONTRIBUTING.md

```diff
@@ -46,9 +46,8 @@ To build the `main` branch:
 ```
 git clone https://github.com/pgopr/pgopr.git
 cd pgopr
-cargo build
-cd target/debug
-./pgopr
+make build
+pgopr
```

Comment:
well, the binary won't be installed, so you need the `cd` and `./pgopr`

Reply:
ha true, I missed that